usage_examples/streamlit/streamlit_app.py

```diff
@@ -0,0 +1,32 @@
+from spacy import util
+from spacy_streamlit import visualize_ner, visualize_textcat
+import streamlit as st
+
+DEFAULT_TEXT = "Matt Honnibal is the CEO of Explosion."
+
+st.title("spacy-llm Streamlit Demo")
```

Comment:
Maybe we can add a little intro here with a link to the repo and a short summary of what it is and what the demo does?

Reply:
Is [this](https://github.com/explosion/spacy-llm/blob/2cd47c01e95e2b774042efb655efe7160d539205/usage_examples/streamlit/streamlit_app.py#L11) enough, or should I expand upon anything?